doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -11,11 +11,12 @@ This tutorial has following sections:
 - [Azure SDK Code Generation Quickstart Tutorial (Data Plane)](#azure-sdk-code-generation-quickstart-tutorial-data-plane)
```

Comment:
I am still quite confused where the line is between one time setup and what is needed to generate every time.

It looks like we created a new file `AzureSDKPackage_Setup.md` to handle the one time setup but there is still package setup steps in here which is confusing.

Can we create a clear separation between one time setup and "how to generate"